packages/client/src/object/aggregate.test.ts

```diff
@@ -15,15 +15,17 @@
  */
 
 import type { ObjectTypeDefinition, OntologyDefinition } from "@osdk/api";
+import type {
+  AggregateOpts,
+  AggregateOptsThatErrors,
+  GroupByClause,
+} from "@osdk/client.api";
```

Comment:
I wonder if we should be splitting part of this test out?

Reply:
Can look in a FLUP, bryant's PR also touches this a lot so can wait until after that